beagle_etl/admin.py

```diff
@@ -18,9 +20,26 @@ def restart(modeladmin, request, queryset):
 
 restart.short_description = "Restart"
 
+class RecipeFilter(SimpleListFilter):
+    title = 'Recipe'
+    parameter_name = 'recipe'
 
-class JobAdmin(admin.ModelAdmin):
-    list_display = ('id', 'run', 'retry_count',
+    def lookups(self, request, model_admin):
+        options = set()
+
+        for o in Operator.objects.values("recipes"):
+            for recipe in o["recipes"]:
+                options.add((recipe, recipe))
+        return options
```

Comment:
a caveat is that it doesn't catch all recipes coming from LIMS, just those used in Operators